ext/mysqli/mysqli.stub.php

```diff
@@ -474,7 +474,7 @@ public function num_rows() {}
      * @return bool
      * @alias mysqli_stmt_send_long_data
      */
-    public function send_long_data(int $param_nr, string $data) {}
+    public function send_long_data(int $index, string $data) {}
```

Comment:
I think the meaning of `$index` in this context is not very obvious. I'd retrain `$param_num` as the name here.

Reply:
I don't even understand what this method does, so I don't know what makes most sense. :smile: